### PR TITLE
Add Atomic operation support

### PR DESCRIPTION
**Pull request by waxzce**
to demonstrate, just launch a couchbase on localhost with default bucket, and run
testOnly AtomicTest
in the sdb shell

### CODE REVIEW

File: plugin/src/test/scala/AtomicTest.scala [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import scala.concurrent.duration.Duration
+import java.util.UUID
+import akka.pattern.after
+import scala.concurrent.duration._
+import play.api.libs.concurrent.Akka
+import scala.concurrent.Future
+import scala.util.Success
+import scala.util.Failure
+
+class AtomicTest extends Specification {
```
Comment (mathieuancelin) [inline plugin/src/test/scala/AtomicTest.scala:22]: WUT ??? Tests ??? that's awesome ;-)

Comment (waxzce) [inline plugin/src/test/scala/AtomicTest.scala:22]: :-)

Comment (waxzce) [inline plugin/src/test/scala/AtomicTest.scala:22]: in this case, tests are not really usefull, but I will rewrite it soon :-)

Comment (mathieuancelin) [inline plugin/src/test/scala/AtomicTest.scala:22]: I have some stuff to rewrite too ;-)

**Comment by waxzce** [comment]
je devais régler un problème

**Comment by waxzce** [comment]
any news ?

**Comment by mathieuancelin** [comment]
Hi Quentin,

It's just pretty hard for me to find free time to merge the PR.

I'll do it ASAP

**Comment by mathieuancelin** [comment]
Should I merge #31 or #32 ?

**Comment by waxzce** [comment]
both :-)

but #32 introduce a small regression to existing code, you have to import something every time you use a function with expiration

**Pull request merged**